recipes/cp2k/2025.1/gh200/environments.yaml

```diff
@@ -65,3 +68,70 @@ cp2k:
       exclude: ["cp2k"]
     cp2k:
       link: roots
+cp2k-dlaf:
+  compiler:
+  - toolchain: gcc
+    spec: gcc@12.3
+  unify: true
+  specs:
+  - cmake
+  - ninja
+  - cuda@12.4 # Can't be newer than driver version, for DBCSR
+  - openblas threads=openmp
+  - netlib-scalapack
+  - fftw
+  - libxc@7 ~cuda build_system=cmake
+  - spglib
+  - spla
+  - elpa
+  - plumed
+  # libint
+  - libint
+  - eigen@3
+  # pika
+  - pika@0.30.1 malloc=mimalloc
+  - mimalloc
+  - boost
+  - fmt
+  - spdlog
+  - hwloc
+  - whip
+  # Umpire
+  - umpire
+  - blt
+  - camp
+  # DLA-Future
+  - dla-future@0.7.3 +mpi_gpu_aware +mpi_gpu_force_contiguous +scalapack +miniapps
```

Comment:
Do we want to bump the dla-future, dla-future-fortran, and pika versions at the same time or keep them the same (c.f. https://github.com/eth-cscs/alps-uenv/pull/197)? Leaving them unchanged might be easier and we bump them for the 2025.2 uenv?

Reply:
I could bump them, but I wanted to keep the environment as similar as possible. Maybe worth doing the update with 2025.2? (It should be out in the next month or so...)

Reply:
Yep, sounds like less trouble to bump them for 2025.2.